app/charts/shared/use-sync-interactive-filters.tsx

```diff
@@ -44,14 +49,35 @@ const useSyncInteractiveFilters = (chartConfig: ChartConfig) => {
 
   // Time slider filter
   const timeSliderFilter = interactiveFiltersConfig?.timeSlider;
+  const xComponentIri = getFieldComponentIri(chartConfig.fields, "x");
   useEffect(() => {
     if (
       timeSliderFilter?.componentIri === "" &&
       IFstate.timeSlider.value !== undefined
     ) {
       dispatch({ type: "RESET_TIME_SLIDER_FILTER" });
     }
-  }, [IFstate.timeSlider.value, timeSliderFilter?.componentIri, dispatch]);
+
+    if (
+      xComponentIri !== undefined &&
+      xComponentIri === timeSliderFilter?.componentIri
+    ) {
+      dispatchConfigurator({
+        type: "INTERACTIVE_FILTER_CHANGED",
+        value: {
+          ...interactiveFiltersConfig,
```

Comment:
Nit: Maybe there could be a "EMPTY_TIME_SLIDER_FILTER" action type, to be coherent with what's above ? I do not like that we have to pass the interactiveFiltersConfig here, I would prefer that this logic is in the reducer.